src/service/url-replacements-impl.js

```diff
@@ -352,6 +352,21 @@ export class GlobalVariableSource extends VariableSource {
     // Returns the user's time-zone offset from UTC, in minutes.
     this.set('TIMEZONE', dateMethod('getTimezoneOffset'));
 
+    // Returns the IANA timezone code
+    this.set('TIMEZONE_CODE', () => {
+      let tzCode = '';
+      if (Intl && typeof Intl === 'object') {
+        tzCode = new Intl.DateTimeFormat().resolvedOptions().timeZone;
+
+        // It could be undefined (i.e. IE11)
```

Comment:
Good catch! 
One question I have: when getting `timeZone` undefined in IE. Will it throw an error? Or it returns undefined silently? 

BTW: You can save this by returning `tzCode ? tzCode : ''`